homework-submission/HomeWork/main.js

```diff
@@ -0,0 +1,73 @@
+// create the Header component
+class Header extends React.Component {
```

Comment:
Super to a reusable component for the header; this would be a perfect case for a functional (presentational) component though :)

Reply:
Thanks a lot 👍